data-explorer/kusto/query/activity-counts-metrics-plugin.md

```diff
@@ -26,7 +26,7 @@ T | evaluate activity_counts_metrics(id, datetime_column, startofday(ago(30d)),
 | *TimelineColumn* | string | &check; | The name of the column that represents the timeline. |
 | *Start* | datetime | &check; | The analysis start period. |
 | *End* | datetime | &check; | The analysis end period. |
-| *LoockbackWindow* | decimal, datetime, or timespan | &check; | The analysis window period. The value may also be a string of `week`, `month`, or `year`, in which case all periods would be [startofweek](startofweekfunction.md), [startofmonth](startofmonthfunction.md), or [startofyear](startofyearfunction.md). |
+| *LookbackWindow* | decimal, datetime, or timespan | &check; | The analysis window period. The value may also be a string of `week`, `month`, or `year`, in which case all periods would be [startofweek](startofweekfunction.md), [startofmonth](startofmonthfunction.md), or [startofyear](startofyearfunction.md). |
```

Comment:
should be called either `Step` or `Window` like in `new_activity_metrics` or `activity_metrics`. Lookback is not the right term to use.

Reply:
@yifatsor substituted ```LookbackWindow``` by ```Window```